src/utils/hints/tokensHints.js

```diff
@@ -1,5 +1,17 @@
 export const tokenHints = {
   tokenSymbol: 'Non-unique symbol of the token.',
   tokenName: 'Full name of the token.',
-  tokenContractId: 'Identifier of a the Smart Contract that implements AEX-9 interface.',
+  tokenContractId: 'Identifier of the Smart Contract that implements AEX-9 interface.',
+  price: 'Fiat price of one token.',
+  marketCap: 'Total fiat market value of all tokens.',
```

Comment:
I would suggest:
`Total market value of a token calculated by multiplying the current price by the total circulating supply.`